### ISSUE TITLE
Problem with pagination and batch create

### ISSUE THREAD

**Issue by guycall** [activescaffold/active_scaffold#570]
Hi, I have a problem with pagination, which I am not sure if it relates to `active_scaffold` or `active_scaffold_batch`. I am running the following gem versions:
* rails (4.2.10)
* active_scaffold (3.4.41.1)
* active_scaffold_batch (3.4.0)

I have the following relevant models:
```
class User < ActiveRecord::Base
  has_many :trainings
   ...
end

class Training < ActiveRecord::Base
  belongs_to :unit
  belongs_to :user
  ...
end

class Unit < ActiveRecord::Base
  has_many :trainings
end
```

And the following controllers
```
class UsersController < ApplicationController
  active_scaffold :user do |config|
    ...
  end
end

class TrainingsController < ApplicationController
  active_scaffold :training do |config|
    config.actions.add :batch_create
    config.batch_create.default_batch_by_column = :unit
    ...
  end
end
```

If I visit `http://localhost:3000/users` and then view the trainings of any user with lots of trainings, I get the pagination links for the trainings. The pagination works correctly. If I look in `_list_pagination_links.html.erb` the value of `pagination_url_options` is:
```
{:association=>"trainings", :parent_scaffold=>"users", :user_id=>"15971", :utf8=>"✓", :controller=>"/trainings", :action=>"index"} 
```

However, if I then use batch create to make any number of new trainings, and then try to use the pagination I get an error. I see in `_list_pagination_links.html.erb` that the value of `pagination_url_options` is now seems to have action incorrectly set to `batch_create`:
```
{:association=>"trainings", :parent_scaffold=>"users", :user_id=>"15971", :utf8=>"✓", :controller=>"/trainings", :action=>"batch_create"}` 
```
and the pagination links are now incorrectly pointing to: `http://localhost:3000/trainings/batch_create?association=trainings&page=2&parent_scaffold=users&user_id=15971`

**Comment by scambra** [comment]
fixed on master 6d16772241d088e9ef7e4594ea226ea37c5a084c
backported to 3-4-stable b1b85d06 and released 3.4.42

**Comment by guycall** [comment]
Thank you - I checked and now working 🙇
